### PR TITLE
fix: Crashing in linux for copy-paste.

### PR DESCRIPTION
**Pull request by rcshubhadeep**
Quick fix for now. I had left C a while ago. And I am still having some trouble to get back to it. 

How do you actually debug an application like `e` 

Using Python all these times had taken its toll on me :(

### CODE REVIEW

File: src/editor.c [hunk 1]
```diff
@@ -510,16 +510,20 @@ e_context* e_initial(e_context* ctx, int c) {
...
     case 'v': {
       char* str = e_clipboard_paste();
-      e_context* new = e_context_copy(ctx);
-      new->history = ctx;
-      int i = 0;
-      while (str[i] != '\0') {
-        if (str[i] == '\n' || str[i] == '\r') e_insert_newline(new);
-        else e_insert_char(new, str[i]);
-        i++;
+      if (str != NULL){
```
Comment (hellerve) [inline src/editor.c]: Instead you could write `if (!str) break;` as an early exit. That avoids wrapping everything in a big `if` statement.

**Review by dcousens — APPROVED** [review]

**Comment by rcshubhadeep** [comment]
Done :)

**Comment by hellerve** [comment]
Perfect! Thank you!

**Comment by rcshubhadeep** [comment]
Welcome :) Thanks for this nice editor also. happy to contribute

**Pull request merged**